src/dmd/parse.d

```diff
@@ -4468,7 +4494,7 @@ final class Parser(AST) : Lexer
                         // ref (parameters) { statements... }
                         // ref (parameters) => expression
 
-                        AST.Dsymbol s = parseFunctionLiteral();
+                        s = parseFunctionLiteral();
```

Comment:
I think Walter doesn't like reusing variables and prefers declaring new ones.

Reply:
that's right.

Reply:
It's not reused. It's assigned only once in each branch handling a particular form of alias.